_posts/2019/2019-01-13-use-multiple-migrations-when-adding-database-constraints.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Use Multiple Migrations When Adding Database Constraints
+description: You need to avoid a long table rebuild and a write-lock
+layout: article
+category: ruby
+image:
+  base: '2018/use-multiple-migrations-when-adding-database-constraints'
+  alt: 'Locks'
+  source: 'https://unsplash.com/photos/8_NI1WTqCGY'
+  credit: 'marcos mayer'
+---
+
+Adding constraints to your application at the database layer is a good idea as it gives an extra layer of quality control to the data powering your application.
```

Comment:
*_on top of_ the data powering your application (?)

And I'd say 'provides' instead of 'gives' an extra layer...